### ISSUE TITLE
OBS rendering problem when dragging between two monitors if scaling is enabled on one of them.

### ISSUE THREAD

**Issue by GloriousEggroll** [obsproject/obs-studio#8910]
### Operating System Info

Other

### Other OS

Nobara, Fedora

### OBS Studio Version

29.1.1

### OBS Studio Log URL

https://obsproject.com/logs/RfLd6wUVldOBBslj

### Expected Behavior

I should be able to drag obs between my monitors without it's GUI rendering breaking.

### Current Behavior

When I drag OBS from the display it was opened on to the other OBS's GUI rendering breaks:

https://youtu.be/3dCI_3HFi8g

### Steps to Reproduce

This issue occurs on both GNOME Mutter in wayland as well as labwc (an openbox-like wayland compositor, similar to hyprland or sway). On GNOME you have to enable experimental fractional scaling https://www.omglinux.com/how-to-enable-fractional-scaling-fedora/ then in display settings set one monitor to something anywhere between 125-175% (not 100 or 200), on labwc you can use 'wdisplays' application to set a fractional scaling value (like 150% or 175%). I have not had this issue occur on any other applications except OBS.

Once fractional scaling is set, open OBS and try dragging it from one display to another.

**Comment by kkartaltepe** [comment]
I dont think we have any custom (rendering) widgets in the settings. Seeing how that window break it looks like a Qt bug to me.

**Comment by GloriousEggroll** [comment]
FWIW it also happens on the flatpak version

**Comment by GloriousEggroll** [comment]
This definitely has something to do with Qt's scaling. After reading docs here:

https://doc.qt.io/qt-6/highdpi.html

I launched obs using:
QT_SCREEN_SCALE_FACTORS=1.5 obs

and using a value that matches my fractional scaling (1.5 = 150%) allows moving from one screen to another without breaking rendering on the main window, however subwindows (such as settings) are still broken.

**Comment by SuslikV** [comment]
Reminds me the https://obsproject.com/forum/threads/exiting-studio-mode-causes-rendering-problem.55364/
was it helpful?

**Comment by GloriousEggroll** [comment]
> Reminds me the https://obsproject.com/forum/threads/exiting-studio-mode-causes-rendering-problem.55364/ was it helpful?

unfortunately no this doesnt resolve the issue

**Comment by SuslikV** [comment]
`xdpyinfo | grep DRI` output?

other graphics card (not amd)?

**Comment by kkartaltepe** [comment]
#9004 suggests this may be a regression in Qt 6.5, as they reported everything working with flatpak on Qt 6.4

**Comment by dPing** [comment]
This same scaling issue happens on KWIN (plasma desktop) as well.  the new change to KWIN is enabling fractional scaling so I believe this is the same issue
